hoot-core/src/main/cpp/hoot/core/schema/TagComparator.cpp

```diff
@@ -524,8 +526,13 @@ void TagComparator::mergeNames(Tags& t1, Tags& t2, Tags& result)
     }
     else if (result.contains(it2.key()))
     {
-      QStringList sl = Tags::split(it2.value());
-      altNames.insert(sl.begin(), sl.end());
+      const Qt::CaseSensitivity caseSensitivity =
+        _caseSensitive ? Qt::CaseSensitive : Qt::CaseInsensitive;
+      if (result[it2.key()].compare(it2.value(), caseSensitivity) != 0)
+      {
```

Comment:
Enumerated tags are different from the freeform text names. E.g. "name=Joe's Crab Shack" vs "amenity=restaurant". It is arguable as to whether enumerations should ever be case senstive, but I don't think they should be controlled by a single config option. We should chat about it.